docs/README.rst

```diff
@@ -1,12 +1,55 @@
-====
-salt
-====
-
-.. image:: https://travis-ci.com/saltstack-formulas/salt-formula.svg?branch=master
-    :target: https://travis-ci.com/saltstack-formulas
+.. _readme:
+
+salt-formula
+============
+
+|img_travis| |docs| |img_sr|
+
+.. |img_travis| image:: https://travis-ci.com/saltstack-formulas/salt-formula.svg?branch=master
+   :alt: Travis CI Build Status
+   :scale: 100%
+   :target: https://travis-ci.com/saltstack-formulas/salt-formula
+.. |docs| image:: https://readthedocs.org/projects/docs/badge/?version=latest
+   :alt: Documentation Status
+   :scale: 100%
+   :target: https://salt-formula.readthedocs.io/en/latest/?badge=latest
+.. |img_sr| image:: https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg
+   :alt: Semantic Release
+   :scale: 100%
+   :target: https://github.com/semantic-release/semantic-release
 
 Yes, Salt can Salt itself!
 
+.. contents:: **Table of Contents**
+
+General notes
+-------------
+
+See the full `SaltStack Formulas installation and usage instructions
+<https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html>`_.
+
+If you are interested in writing or contributing to formulas, please pay attention to the `Writing Formula Section
+<https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#writing-formulas>`_.
+
+If you want to use this formula, please pay attention to the ``FORMULA`` file and/or ``git tag``,
+which contains the currently released version. This formula is versioned according to `Semantic Versioning <http://semver.org/>`_.
+
+See `Formula Versioning Section <https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#versioning>`_ for more details.
+
+Contributing to this repo
+-------------------------
+
+**Commit message formatting is significant!!**
+
+Please see :ref:`How to contribute <CONTRIBUTING>` for more details.
+
+Available states
+----------------
```

Comment:
While this `Available states` is correct including the heading symbols, the existing duplicate below (line 59) needs to be removed.